apps/arena/lib/arena/game_updater.ex

```diff
@@ -105,7 +106,14 @@ defmodule Arena.GameUpdater do
         }
 
         state =
-          put_in(state, [:game_state, :players, player_id, :aditional_info, :bounties], bounties)
+          update_in(state, [:game_state, :players, player_id, :aditional_info], fn aditional_info ->
+            if Enum.empty?(bounties) do
+              aditional_info
+            else
+              aditional_info
+              |> Map.put(:bounties, bounties)
+            end
+          end)
```

Comment:
Can you explain a bit on why is this needed or what it is doing?

Reply:
Whoops, this was something i needed to pick the `default_bounty` since if the bounties list is empty the `Enum.random/1` method would crash